src/Resolver/Reflector.php

```diff
@@ -129,7 +129,7 @@ public function getTraits($class)
             } while ($class = get_parent_class($class));
 
             // get traits from ancestor traits
-            while (list($trait) = each($traits)) {
+            foreach($traits as &$trait) {
```

Comment:
Thank you for the information.

Why did you used `&` ? 

It may be good if you can add a space between `foreach` and `(` as `foreach ($traits` .

Not sure why the tests fails, will look into it.

Thank you.